tools/user_parts_clone.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+#
+# enable user to override installed parts library with GitHub clone
+
+REPODIR="${HOME}/.local/share/fritzing/fritzing-parts"
+mkdir -p "${REPODIR}"
+git clone --branch master --single-branch https://github.com/fritzing/fritzing-parts.git "${REPODIR}" || exit 1
+Fritzing -db "${REPODIR}/parts.db"
```

Comment:
```suggestion
Fritzing -parts "${REPODIR}"
```

Reply:
This line is about generating parts.db, not starting Fritzing. I think that is OK?
I can split this commit off into a separate pull request if that would be useful.

Reply:
A script to switch part repos quick would certainly be useful for parts development, so yes, you could split that out (maybe even move to the fritzing-parts repository) If you just change the part files, not the location (like 'git checkout') then updating the -db would be needed. But I assume, when you run 'git clone', that you now have a directory that was previously not there, and to tell fritzing about this new directory, you would need the -path option, not -db.

Reply:
OK. Makes sense. I think it needs a conclusion on what is going on with the directories before moving forward with this script - `~/.local/share/fritzing` would be supported at present.